src/commands/ls_remote.rs

```diff
@@ -1,24 +1,56 @@
 use crate::config::FnmConfig;
 use crate::remote_node_index;
+use crate::user_version::UserVersion;
+
+use colored::Colorize;
 use thiserror::Error;
 
 #[derive(clap::Parser, Debug)]
-pub struct LsRemote {}
+pub struct LsRemote {
+    /// Filter for versions
+    filter: Option<UserVersion>,
+
+    /// Only show latest LTS versions
+    #[arg(long)]
+    lts: Option<Option<String>>,
+
+    // Version sorting order
+    #[arg(long, default_value = "asc")]
+    sort: remote_node_index::SortingMethod,
```

Comment:
This one just has a regular comment instead of a doc comment, typo?

```suggestion
    /// Sort versions output
    #[arg(long, default_value = "asc")]
    sort: remote_node_index::SortingMethod,
```